### PR TITLE
Cc among us project

### PR DESCRIPTION
**Pull request by Ingrammyles8**
Among Us Community Contribution

### CODE REVIEW

File: among_us_stats.Rmd [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+library(dplyr)
+library(tidyverse)
+library(plyr)
+require(scales)
+```
+
+This R code runs player level and community level statistics on Among Us Player data (which can be found on the home page of the app). To use this script, add your name and a userid to the among_us_player_stats.csv along with your player stats. Then enter your username and userID into the code, run the script and take a look! This code will use my username as an example.
+```{r}
+plyr_stats <- read_csv("../resource/among_us_player_stats.csv")
+Username = "Big Cheese"
```
Comment (lubin-liu) [inline among_us_stats.Rmd:19]: Please double check this link, I think it should be 

resources/among_us_stats/among_us_player_stats.csv

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# cc19
```
Comment (lubin-liu) [inline README.md]: I will ask Prof. Robbins to check this whether you should be updating the README

**Review by lubin-liu — CHANGES_REQUESTED** [review]
Please see the comments below. Also, please make sure that the branch name and .Rmd file names match.

**Comment by jtr13** [comment]
Thanks for checking... the date has already been updated in the README.md so best to remove the README.md from this pull request.

File: README.md (hunk 2 above)
Comment (lubin-liu) [reply README.md]: Per comment below, please remove the README from the pull request.

**Review by lubin-liu — CHANGES_REQUESTED** [review]
Please remove the README from the pull request

File: README.md (hunk 2 above)
Comment (Ingrammyles8) [reply README.md]: How do I remove a file from a pull request? Do I need to submit another PR?

Comment (lubin-liu) [reply README.md]: @jtr13 Can you advise?

Comment (jtr13) [reply README.md]: Go back to your local branch `cc_among_us_project` and delete the README file. Then commit and push again and the PR will be automatically updated.

File: among_us_stats.Rmd (hunk 1 above)
Comment (lubin-liu) [reply among_us_stats.Rmd:19]: See above

**Review by lubin-liu — CHANGES_REQUESTED** [review]
Please double check the link on line 18, I think it should be

resources/among_us_stats/among_us_player_stats.csv

**Comment by lubin-liu** [comment]
@jtr13  It seems like github is not allowing me to resolve conflicts?

**Comment by jtr13** [comment]
Strange... apparently the resolve conflicts button gets disabled if the merge conflict is "too complex" to resolve on GitHub, though I don't know why that would be the case here. I'll try to resolve it locally.
https://stackoverflow.com/questions/59110548/github-resolve-conflicts-button-disabled

**Comment by jtr13** [comment]
@lubin-liu The Resolve conflicts button is back--it has all the .Rmds so you just need to remove the >>>> <<<<< lines.

**Comment by lubin-liu** [comment]
Your contribution has been merged! Thanks! 🎉

**Pull request merged**